lib/stages/visTranspose.cpp

```diff
@@ -266,10 +332,11 @@ void visTranspose::main_thread() {
         }
 
         // Increment within read chunk
-        ti = (ti + 1) % write_t;
-        if (ti == 0)
-            fi++;
-        if (fi == write_f) {
+        // within a chunk, frequency is the fastest varying index
```

Comment:
Can you explain this change?

Reply:
this is the complement to the change in the incoming order in `visRawReader`